### DISCUSSION TITLE
DetailsList in V9

### DISCUSSION THREAD

**Discussion by jamesongit** [microsoft/fluentui#25017] [Q&A] [answered]
Hi,
No Detailslist planned for v9 in roadmap, will it be added later or another replacement?

Thanks

**Comment by ValentinaKozlova** [comment, marked as answer]
Hello @jamesongit, currently there's a DataGrid component: https://react.fluentui.dev/?path=/docs/components-datagrid--docs
This is a V9 version of Detaild list (Migration docs: https://react.fluentui.dev/?path=/docs/concepts-migration-from-v8-component-mapping--docs)
